mobify/platform-scaffold
# Unify country and region fields

I discovered in preparing to rebuild the schema for checkout locations that the usage of the location (country and region) data was inconsistent between the three places it was used: cart shipping estimate, shipping address, and billing address. This PR splits those out into their own components, so that the logic can be shared and consistent.

 **JIRA**: Preparatory for checkout schema
 **Linked PRs**: (links to corresponding PRs, optional)

## Changes
- 

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to the cart, and open the cart
- Check that the shipping estimate modal and form work correctly
- Check that the shipping address form works correctly
- Check that the billing address form works correctly
